src/main/java/org/opensearch/index/codec/customcodecs/QatDeflate99Codec.java

```diff
@@ -86,6 +86,9 @@ public boolean supports(Setting<?> setting) {
 
     @Override
     public Set<String> aliases() {
+        if (!QatZipperFactory.isQatAvailable()) {
```

Comment:
This isn’t called anywhere in a loop is it?

Reply:
Seems like this was used only in the tests (and most of the tests were ignored due to this check :( )